htdocs/install/mysql/migration/9.0.0-10.0.0.sql

```diff
@@ -254,3 +254,6 @@ ALTER TABLE llx_expensereport_det ADD COLUMN subprice double(24,8) DEFAULT 0 NOT
 ALTER TABLE llx_product_attribute_combination ADD INDEX idx_product_att_com_product_parent (fk_product_parent);
 ALTER TABLE llx_product_attribute_combination ADD INDEX idx_product_att_com_product_child (fk_product_child);
 
+ALTER TABLE llx_product ADD COLUMN fk_project integer, ADD INDEX fk_project (fk_project);
```

Comment:
You must use 2 request, on to add column and one to add the index.
You must also put this information into the definition of table llx_product.sql and llx_product.key.sql